### PR TITLE
dcp19-header-footer: Made UI Changes for the header and footer region.

### PR DESCRIPTION
**Pull request by theyogendraDanwar**

### CODE REVIEW

File: docroot/themes/custom/dcp/dcp.libraries.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ global-styling:
   css:
     base:
       css/style.css: {}
-      single-page/style.css: {}
+#      single-page/style.css: {}
```
Comment (prashant114606-zz) [inline docroot/themes/custom/dcp/dcp.libraries.yml:6]: any reason? why are we commenting it for now!! this css is being used for single page site

Comment (theyogendraDanwar) [reply docroot/themes/custom/dcp/dcp.libraries.yml:6]: @prashant114606 this is conflicting with the top style.css file

File: docroot/themes/custom/dcp/js/main.js [hunk 2]
```diff
@@ -26,4 +26,23 @@
...
 
+  Drupal.behaviors.addActiveLink = {
+    attach: function (context, settings) {
+      $( window ).resize(function() {
+        if(window.innerWidth > 992) {
+          $('body').css('overflow', 'auto');
+          $('.navigation-mobile').hide();
+        }
+      });
+      $('.region-header .mobile-menu.trigger-icon').on('click',() => {
```
Comment (prashant114606-zz) [inline docroot/themes/custom/dcp/js/main.js:37]: condition missing!! should be applied only for mob and tab no?

Comment (theyogendraDanwar) [reply docroot/themes/custom/dcp/js/main.js:37]: @prashant114606  can you provide me which specific line you are referring to

Comment (prashant114606-zz) [reply docroot/themes/custom/dcp/js/main.js:37]: these are for hamburger menu right?
and those are only on mob + tab

File: docroot/themes/custom/dcp/dcp.libraries.yml (hunk 1 above)
Comment (prashant114606-zz) [reply docroot/themes/custom/dcp/dcp.libraries.yml:6]: this needs a discussion!! let's discuss this in next call
cc @ajitdev @dipakmdhrm

File: docroot/themes/custom/dcp/js/main.js (hunk 2 above)
Comment (theyogendraDanwar) [reply docroot/themes/custom/dcp/js/main.js:37]: yes this is only for mob and tab

File: docroot/themes/custom/dcp/sass/components/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.region-footer {
+  min-height: 390px;
+  background-image: url("../images/footer_image.png");
+  background-size: 100% 100%;
+  padding: 60px;
+  display: flex;
+
+  [class*="menu--"] {
```
Comment (vidit-anjaria) [inline docroot/themes/custom/dcp/sass/components/_footer.scss:8]: @theyogendraDanwar - can we use menu attribute module?

File: docroot/themes/custom/dcp/sass/components/_header.scss [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+  }
+  img {
+    width: 100%;
+  }
+  #block-logintext {
+    padding-left: 50px;
+    a {
+      background: #FF2370;
+      border-radius: 23.5px;
+      font-family: Circular Air;
```
Comment (vidit-anjaria) [inline docroot/themes/custom/dcp/sass/components/_header.scss]: @theyogendraDanwar - i think, you might have to change the font family as per the fonts added. Please refer this file -  https://github.com/nikitaaswani/dcp/blob/b310857390fc3386178e5a2a08d90299e808487b/docroot/themes/custom/dcp/sass/utilities/_typography.scss

File: docroot/themes/custom/dcp/sass/pages/_homepage.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (vidit-anjaria) [inline docroot/themes/custom/dcp/sass/pages/_homepage.scss]: @theyogendraDanwar - please resolve the conflict.

**Review by vidit-anjaria — COMMENTED** [review]
please make the suggested changes.

File: docroot/themes/custom/dcp/sass/components/_footer.scss (hunk 3 above)
Comment (theyogendraDanwar) [reply docroot/themes/custom/dcp/sass/components/_footer.scss]: @vidit-anjaria this is only for the footer region it doesn't make sense to me add a whole module just for the footer for a small section if in future there are multiple requirements then we can add that

Comment (theyogendraDanwar) [reply docroot/themes/custom/dcp/sass/components/_footer.scss:8]: @vidit-anjaria this is only for the footer region it doesn't make sense to me add a whole module just for the footer for a small section if in future there are multiple requirements then we can add that

**Review by vidit-anjaria — APPROVED** [review]

File: docroot/themes/custom/dcp/sass/components/_footer.scss (hunk 3 above)
Comment (vidit-anjaria) [reply docroot/themes/custom/dcp/sass/components/_footer.scss:8]: @theyogendraDanwar - I would suggest to use the menu attribute module though.

**Pull request merged**